<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>534</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="5">
     <widget class="QPushButton" name="saveButton">
      <property name="toolTip">
       <string>Save the items from list</string>
      </property>
      <property name="text">
       <string>&amp;save</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/png/save.png</normaloff>:/png/save.png</iconset>
      </property>
     </widget>
    </item>
    <item row="5" column="1" rowspan="5" colspan="4">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="hashCounterBox">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>85</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>85</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="toolTip">
       <string>Shows how many items was loaded in the hash</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Current Path:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Filters:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QLineEdit" name="currentPathEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="reloadHashButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reloads the directory in a hash. Use this button when entered new file extension!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/png/rotate.png</normaloff>:/png/rotate.png</iconset>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="clearAllButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>Delete all items from the List</string>
      </property>
      <property name="text">
       <string>clear all</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/png/trash.png</normaloff>:/png/trash.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QCheckBox" name="useHashkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Using hash is pretty faster to find a file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>use hash</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>File:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Items:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="findButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Start to find file</string>
      </property>
      <property name="text">
       <string>&amp;find</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLineEdit" name="filePathEdit">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>85</green>
            <blue>127</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="itemsCounterBox">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="toolTip">
       <string>Items who was found</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="filtersLineEdit">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When using hash, enter the extension of file e.g. *.exe , *.png.&lt;/p&gt;&lt;p&gt;If this line is empty, automaticly *.exe is the filter&lt;/p&gt;&lt;p&gt;And press the reload button.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="placeholderText">
       <string> *.exe , *.png ...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>File path:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>&amp;close</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QCheckBox" name="checkAllBox">
      <property name="toolTip">
       <string>Check if more than one file exist.</string>
      </property>
      <property name="text">
       <string>check all</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="fileLineEdit">
      <property name="placeholderText">
       <string>Entere here the file to find</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QToolButton" name="deleteItemButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/png/trash.png</normaloff>:/png/trash.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QToolButton" name="openButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Try to open file</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/png/openExternal.png</normaloff>:/png/openExternal.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
